public/dashboard.js

```diff
@@ -21,6 +20,24 @@ let tokenExpiryTime = -1;
 let serverRunning = false;
 let notyf;
 
+let apiBase;
+getApiBase = async () => {
+    console.log("Called getApiBase")
```

Comment:
if we need this for debugging make it `console.debug`, but perhaps it's not needed?